webknossos/webknossos/dataset/layer.py

```diff
@@ -691,7 +664,15 @@ def add_existing_remote_mag_view(
             f"Cannot add mag {mag} as it already exists for layer {self}"
         )
         self._setup_mag(mag, mag_path)
-        self._properties.mags.append(mag_view._properties)
+        # since the remote mag view might belong to another dataset, it's property's path might be None, therefore, we get the path from the mag_view itself instead of it's properties
+        self._properties.mags.append(
+            MagViewProperties(
+                mag=mag_view.mag,
+                path=str(mag_view.path),
```

Comment:
Just casting `str(..)` probably doesn't work, because wk expects a different syntax for S3 paths, i.e. `s3://{endpoint_url/{bucket_name}/{prefix}` instead of `s3://{bucket_name}/{prefix}`